rebuild-amplify/rebuild.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
can we add comments this is an alternative script to TS version

Reply:
sgtm